roles/nginxplus/files/conf/http/ojs-staging.conf

```diff
@@ -4,6 +4,7 @@ proxy_cache_path /data/nginx/ojs-staging/NGINX_cache/ keys_zone=ojs-stagingcache
 upstream ojs-staging1 {
     zone ojs-staging 64k;
     server ojs-staging1.princeton.edu:443;
+    #server ojs-staging1.princeton.edu:443;
```

Comment:
Should this be ojs-staging2?

Also, should it be uncommented?

Reply:
It should be ojs-staging2 and it is commented because this will be deployed as A/B